docs/codeql/codeql-cli/analyzing-databases-with-the-codeql-cli.rst

```diff
@@ -214,6 +214,11 @@ To analyze your database using the `cpp-security-and-quality.qls` query suite fr
     codeql database analyze --format=sarif-latest --output=results <db> \
        'codeql/cpp-queries@~0.0.3:codeql-suites/cpp-security-and-quality.qls'
 
+If you need to reference a query file, directory, or suite whose path contains a literal `@` or `:`, you can prefix the query specification with `path:` like so::
+
+    codeql database analyze --format=sarif-latest --output=results <db> \
+        path:security/queries@experimental/query.ql
```

Comment:
I suggest an example that doesn't look like a pack-based query spec. Something like `C:/Users/ci/workspace@2/security/query.ql` ?

Reply:
Sure.